src/pages/studies/index.js

```diff
@@ -1,10 +1,110 @@
 import matter from 'gray-matter';
-import Link from 'next/link';
+// import Link from 'next/link';
```

Comment:
Commented out this import since we're not using it anymore — let me know if I should remove it altogether? Kept it in just in case we want to go back to the dynamic posts list in the future.

Reply:
Hey Prabhav, let's keep it for now